packages/components/src/components/real-wallet-card/real-wallet-card.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import RealWalletCardContent from './real-wallet-card-content.jsx';
+import RealWalletCardHeader from './real-wallet-card-header.jsx';
+import RealWalletCardFooter from './real-wallet-card-footer.jsx';
+
+const RealWalletCard = ({
+    amount,
+    currency,
+    getWalletLabels,
+    has_footer = true,
+    has_no_funds = false,
+    is_actions_footer = false,
+    is_deposit_footer = false,
+    is_linked = false,
+    is_selected = false,
+    is_temporarily_unavailable = false,
+    onClickDeposit,
+    onClickSettings,
+    onClickTransactions,
+    onClickTransfer,
+    onClickWithdrawal,
+    wallet_name,
+    width = '280',
+}) => {
+    const [should_show_footer_actions, setShouldShowFooterActions] = React.useState(false);
+    const height = (width / 5) * 3;
+
+    // Get wallet type (first word) before space or forward slash for background and icon
+    const wallet_type = wallet_name.match(/(^[^\/ || ^\s]+)/)[0];
+
+    const onMouseEnter = () => {
+        setShouldShowFooterActions(true);
+    };
+
+    const onMouseLeave = () => {
+        setShouldShowFooterActions(false);
+    };
+
+    return (
+        <React.Fragment>
```

Comment:
No need for this `Fragment`.

Reply:
Remnants of experimenting with something. Thanks!